src/index.html

```diff
@@ -22,6 +22,358 @@
+            src="images/imac.jpeg"
+            alt="iMacA1419"
+          />
+          <h2 class="product__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code">Product code: 195434</p>
+          <div class="product__rating">
+            <ul class="stars stars--count-4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="product__review-count">Reviews: 5</p>
+          </div>
+          <p class="product__price">
+            <span class="product__price-label">Price:</span>
+            <span class="product__price-value">$2,199</span>
+          </p>
+          <a
+            class="product__buy-button"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="product">
+          <img
+            class="product__photo"
+            src="images/imac.jpeg"
+            alt="iMacA1419"
+          />
+          <h2 class="product__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code">Product code: 195434</p>
+          <div class="product__rating">
+            <ul class="stars stars--count-4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="product__review-count">Reviews: 5</p>
+          </div>
+          <p class="product__price">
+            <span class="product__price-label">Price:</span>
+            <span class="product__price-value">$2,199</span>
+          </p>
+          <a
+            class="product__buy-button"
+            href="#"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="product">
+          <img
+            class="product__photo"
+            src="images/imac.jpeg"
+            alt="iMacA1419"
+          />
+          <h2 class="product__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code">Product code: 195434</p>
+          <div class="product__rating">
+            <ul class="stars stars--count-4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="product__review-count">Reviews: 5</p>
+          </div>
+          <p class="product__price">
+            <span class="product__price--label">Price:</span>
```

Comment:
```suggestion
            <span class="product__price-label">Price:</span>
```
Tag doesn't have    [product__price--label]

You cannot use a modifier without the main class in BEM. As it isn't a modifier at all class could be 'product__price-label'